README.md

```diff
@@ -1,46 +1,109 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Tic Tac Toe User Interface
+# Tic Tac Toe The Game
 
 > The purpose of this project is to setup the Tic Tac Toe game, for two players, using Ruby.
 
-- This is the Milestone #3, to add a logic for the game and make it playable.
+ TIC TAC TOE is game for two players represented by a X for one player and a O for the other one. Each player take turns marking the spaces in a 3×3 grid. The main goal is to place three of their marks in a horizontal, vertical, or diagonal row, the one to do this first is the winner. For additional information go to [TicTacToeWiki](https://en.wikipedia.org/wiki/Tic-tac-toe)
+
+
+## Rules for the Tic Tac Toe Game
+![](./img/Tic-tac-toe-animated.gif)
+
+- The game is played on a grid that's 3 squares by 3 squares.
+
+- You are X, your friend (or the computer in this case) is O. Players take turns putting their marks in empty squares.
+
+- The first player to get 3 of her marks in a row (up, down, across, or diagonally) is the winner.
+
+- When all 9 squares are full, the game is over. If no player has 3 marks in a row, the game ends in a tie.
+
 
 ## Built With
 
-- Ruby
+- Ruby,
+
+
+## Getting Started
 
 To get a local copy up and running follow these simple example steps.
 
-## Authors
 
-👤 **Mateo Mojica**
+### Prerequisites
+You need to have Ruby installed in your machine
+
```

Comment:
- [ ] **OPTIONAL** Or users could use an online environment so that they don't have to install anything. You can use this link that is generated with your repo and branch :)
  https://gitpod.io/github.com/tadeuasarro/TicTacToe/tree/readme_game_instructions
  or use repl.it too